Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RazorTargetTests.cs

```diff
@@ -37,29 +38,79 @@ public class RazorTargetTests
 
         public TestContext TestContext { get; set; }
 
+        [TestMethod]
+        public void Razor_RazorSpecificOutputAndProjectInfo_AreCopiedToCorrectFolders()
+        {
+            // Arrange
+            var root = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+            var projectSpecificOutDir = Path.Combine(root, "0");
+            var temporaryProjectSpecificOutDir = Path.Combine(root, "0.tmp");
+            var razorSpecificOutDir = Path.Combine(root, "0.Razor");
+            Directory.CreateDirectory(projectSpecificOutDir);
+            Directory.CreateDirectory(temporaryProjectSpecificOutDir);
+            File.WriteAllText(Path.Combine(projectSpecificOutDir, "ProjectInfo.xml"),
+$@"
+<ProjectInfo>
```

Comment:
Let's throw this whole thing away and use object oriented approach - create new ProjectInfo and `.Save(..)` it to a file. Something like this (except the unneeded `()`)
https://github.com/SonarSource/sonar-scanner-msbuild/blob/45df416d2d4d938a9f655d045e16e1f817bdb35b/Tests/SonarScanner.MSBuild.Shim.Tests/PropertiesFileGeneratorTests_MultiTarget.cs#L271-L288

Dealing with current ProjectInfo XML structure is not purpose of this test.